programmers/알고리즘_3기/2주차/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,20 @@
+function solution(n) {
+  let newN = String(n).split("").reverse();
+  const arr = [];
+
+  newN.forEach((element) => arr.push(Number(element)));
+  return arr;
```

Comment:
forEach를 이용해 String을 Number로 바꾸어서 배열에 넣어주는 방법보다 map을 이용하면 배열을 선언할 필요가 없어서 코드가 줄고 또한 인자에 +를 붙이면 문자가 정수로 바뀌어서 배열에 저장된답니다!
```suggestion
  return newN.map((n) => +n);
```